Sources/Sentry/include/SentryTransportInitializer.h

```diff
@@ -0,0 +1,28 @@
+//
```

Comment:
Curious: Is `Initializer` a cocoa naming? As it would be Factory in some other OOP languages?

Reply:
Whops yet it should be a factory. Sorry brain freeze 😄